{"trial_id": "0000", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "embedding_size", "default": null, "conditions": [], "min_value": 10, "max_value": 100, "step": 10, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "learning_rate", "default": 0.0001, "conditions": [], "min_value": 0.0001, "max_value": 0.01, "step": null, "sampling": "log"}}, {"class_name": "Float", "config": {"name": "l2_regularization", "default": 1e-06, "conditions": [], "min_value": 1e-06, "max_value": 0.0001, "step": null, "sampling": "log"}}], "values": {"embedding_size": 30, "learning_rate": 0.009312551406265033, "l2_regularization": 2.2008649969080477e-06, "tuner/epochs": 2, "tuner/initial_epoch": 0, "tuner/bracket": 3, "tuner/round": 0}}, "metrics": {"metrics": {"loss": {"direction": "min", "observations": [{"value": [0.07180249691009521], "step": 0}]}, "root_mean_squared_error": {"direction": "min", "observations": [{"value": [0.267528772354126], "step": 0}]}, "val_loss": {"direction": "min", "observations": [{"value": [0.06343602389097214], "step": 0}]}}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"c:\\Users\\LENOVO\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"c:\\Users\\LENOVO\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 261, in _run_and_update_trial\n    self.oracle.update_trial(\n  File \"c:\\Users\\LENOVO\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\keras_tuner\\src\\engine\\oracle.py\", line 108, in wrapped_func\n    ret_val = func(*args, **kwargs)\n  File \"c:\\Users\\LENOVO\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\keras_tuner\\src\\engine\\oracle.py\", line 529, in update_trial\n    trial.metrics.register(metric_name, direction=direction)\n  File \"c:\\Users\\LENOVO\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\keras_tuner\\src\\engine\\metrics_tracking.py\", line 219, in register\n    self.metrics[name] = MetricHistory(direction)\n  File \"c:\\Users\\LENOVO\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\keras_tuner\\src\\engine\\metrics_tracking.py\", line 92, in __init__\n    raise ValueError(\nValueError: `direction` should be one of {\"min\", \"max\"}, but got: <built-in function min>\n"}